### ISSUE TITLE
how to show httpsImage in 3.8.1

### ISSUE THREAD

**Issue by zhizi12** [rs/SDWebImage#1629]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios 
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3..8.1
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by zhizi12** [comment]
in 3.7 version ,I can change code like this to show httpsImage,but in 3.8.1 I can not find the function:

- (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge{
   
    if ([challenge previousFailureCount]== 0) {

        NSURLCredential* cre = [NSURLCredential credentialForTrust:challenge.protectionSpace.serverTrust];
        [challenge.sender useCredential:cre forAuthenticationChallenge:challenge];
    }
    else{
    }
}

**Comment by mythodeia** [comment]
if you use the https url of the image in the demo project does it work?

**Comment by idage** [comment]
you can  set SDWebImageOptions in SDWebImageManager

**Comment by mythodeia** [comment]
please reopen if you still face issues
